helpers/locationHelper.py

```diff
@@ -1,59 +1,57 @@
 from __future__ import annotations
 
 from json import loads
+from typing import TypedDict
 from urllib.request import urlopen
 
+import httpx
+
 import settings
 from common.log import logger
+from helpers import countryHelper
+from objects import glob
 
+API_CALL_TIMEOUT = 3
 
-def getCountry(ip: str) -> str:
-    """
-    Get country from IP address using geoip api
 
-    :param ip: IP address
-    :return: country code. XX if invalid.
-    """
-    try:
-        # Try to get country from Pikolo Aul's Go-Sanic ip API
-        result = loads(
-            urlopen(f"{settings.LOCALIZE_IP_API_URL}/{ip}", timeout=3).read().decode(),
-        )["country"]
```

Comment:
This was a complete suprise - we've been doing a synchronous HTTP call on login this whole time lol